scheduler/plugin/plugins.go

```diff
@@ -70,30 +71,49 @@ func NewRegistry(informerFactory informers.SharedInformerFactory, client clients
 
 // NewPluginConfig converts []v1beta2.PluginConfig for simulator.
 // Passed []v1beta.PluginConfig overrides default config values.
+//
+// NewPluginConfig expects that passed v1beta2.PluginConfig has data of Args in only one of PluginConfig.Args.Raw or PluginConfig.Args.Object.
```

Comment:
```
NewPluginConfig expects that either PluginConfig.Args.Raw or PluginConfig.Args.Object has data in the passed v1beta2.PluginConfig parameter.
If data exists in both PluginConfig.Args.Raw and PluginConfig.Args.Object, PluginConfig.Args.Raw would be ignored since PluginConfig.Args.Object has higher priority.
```